### DISCUSSION TITLE
Is it possible to have both "local" & "outbound-links" scripts enabled?

### DISCUSSION THREAD

**Discussion by OD91** [plausible/analytics#1396] [Self-Hosted Support] [answered]
Seems like in my free test plan I cannot track any outbound-links via the outbound-links script (from my localhost) when having the local script in place as well. Do I miss sth? Any help is highly appreciated. Thanks in advance!

**Comment by metmarkosaric** [comment, marked as answer]
"Local" is a recent feature that's not yet available on self hosted so that's probably why. There should be a new self hosted release at some point later this month hopefully. We release them twice a year.
